lib/OpenQA/WebAPI/Plugin/ObsRsync/templates/ObsRsync_gru_index.html.ep

```diff
@@ -0,0 +1,47 @@
+% layout 'bootstrap';
+% content_for 'head' => begin
+  %= asset 'codemirror.js'
+  %= asset 'codemirror.css'
+% end
+
+% title 'OBS synchronization jobs';
+<h2><%= 'OBS synchronization jobs' %></h2>
+
+<div class="row">
+    <div class="col-sm-12">
+
+        %= include 'layouts/info'
+
+        <table class="table table-striped">
+            <thead>
+                <tr>
+                    <th>Id</th>
+                    <th>Task</th>
+                    <th>Args</th>
+                    <th>Created</th>
+                    <th>Started</th>
+                    <th>Priority</th>
+                    <th>Retries</th>
+                    <th>State</th>
+                    <th>Notes</th>
+                </tr>
+            </thead>
+            <tbody>
+                % for my $id (sort keys %$jobs) {
+                    % my $job = $jobs->{$id};
+                    <tr id=$id>
```

Comment:
It doesn't look like the ID would be printed correctly (with quoting and actually printing the contents of the variable). I guess you can just get rid of the ` id=$id` - or is this ID used anywhere?